admin/config/locales/en.yml

```diff
@@ -3558,6 +3558,11 @@ en:
               other: "Has visited <strong>%{count}</strong> times"
           tags:
             description_html: "Tagged with any of <strong>%{tags}</strong>"
+      segmentables:
+        index:
+          title: Content Active for "%{name}"
+          no_content: There's no content setup specifically for this segment.
+          no_content_instructions_html: Throughout the admin, you can setup personalized experiences for this segment. Anywhere you see an <strong>Active</strong> field, you can click the "by segment" link next to it to make that only for a selection of segments. For content blocks, you can do this in the "Display" tab on the right in the block editing screen.
```

Comment:
Minor: This is a bit confusing. " Anywhere you see an <strong>Active</strong> field, you can click the "by segment" link next to it to make that only for a selection of segments."  - make what?